input/pagecontent/changes.md

```diff
@@ -44,37 +47,49 @@ To help implementers, only the more significant changes are listed here.
     <ul>
       <li>Patient.extension changed to:
           <ul>
-            <li>add Individual Pronouns (<a href="https://jira.hl7.org/browse/FHIR-43719">FHIR-43719</a>) and Person Recorded Sex Or Gender</li>
+            <li>add Individual Pronouns (<a href="https://jira.hl7.org/browse/FHIR-43719">FHIR-43719</a>)</li>
+            <li>add Person Recorded Sex Or Gender (<a href="https://jira.hl7.org/browse/FHIR-43834">FHIR-43834</a>)</li>
             <li>add Australian Veteran Status</li>
             <li>remove genderIdentity extension and replace it with the Individual Gender Identity extension (<a href="https://jira.hl7.org/browse/FHIR-43718">FHIR-43718</a>)</li>
           </ul>
       </li>    
         <li>Added invariant inv-pat-1 to apply Gender Identity Response value set (<a href="https://jira.hl7.org/browse/FHIR-43718">FHIR-43718</a>)</li>
         <li>Added invariant inv-pat-2 to apply Australian Pronouns value set (<a href="https://jira.hl7.org/browse/FHIR-43719">FHIR-43719</a>)</li>
+        <li>Added invariant inv-pat-3 to apply Common AU Recorded Sex or Gender Type value set (<a href="https://jira.hl7.org/browse/FHIR-43834">FHIR-43834</a>)</li>
```

Comment:
state more clearly what this is in the change log, people will have to go read some other page to know what "apply" means. Same with next bullet points.

Reply:
Done.

Reply:
need to swap subject and object, as in, you are binding RSG.type to the value set, also state that is extensible. e.g.:
- Added invariant inv-pat-3 to RSG.type to bind Common AU Recorded Sex or Gender Type (extensible).

Apply to all the other similar invariant statements added across the 3 profiles in the change logs.

Reply:
Done